### PR TITLE
[dom] Adding perftools-LAMMPS (mc/gpu)

### PR DESCRIPTION
**Pull request by kraushm**

### CODE REVIEW

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-11Aug2017-CrayGNU-17.08-cuda-8.0-pat-6.5.1.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+name = 'LAMMPS'
+version = '11Aug17'
+release = 'stable_11Aug2017'
+cudaversion =  '8.0'
+patversion = '6.5.1'
+versionsuffix = '-cuda-%s-pat-%s' % (cudaversion, patversion)
+
+homepage = 'http://lammps.sandia.gov/'
+description = "LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) is a classical molecular dynamics simulation code designed to run efficiently on parallel computers."
```
Comment (jgphpc) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-11Aug2017-CrayGNU-17.08-cuda-8.0-pat-6.5.1.eb]: description = """LAMMPS (Large-scale Atomic/Molecular Massively Parallel
Simulator) is a classical molecular dynamics simulation code designed to run
efficiently on parallel computers."""

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-11Aug2017-CrayGNU-17.08-pat-6.5.1.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+easyblock = 'MakeCp'
+
+name = 'LAMMPS'
+version = '11Aug17'
+release = 'stable_11Aug2017'
+patversion = '6.5.1'
+versionsuffix = '-pat-%s' % patversion
+
+homepage = 'http://lammps.sandia.gov/'
+description = "LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) is a classical molecular dynamics simulation code designed to run efficiently on parallel computers."
```
Comment (jgphpc) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-11Aug2017-CrayGNU-17.08-pat-6.5.1.eb]: same comment

**Comment by jgphpc** [comment]
https://github.com/eth-cscs/production/pull/328  explains why the build failed.

**Comment by victorusu** [comment]
retest this please

**Comment by jgphpc** [comment]
* https://jenkins.cscs.ch/job/TestingEB/560/label=dom/console
`merge pull origin master` ?

**Comment by jgphpc** [comment]
retest this please

**Comment by jgphpc** [comment]
https://jenkins.cscs.ch/job/TestingEB/568/label=dom/console failed. 
I copied the logs in /scratch/snx2000tds/jenscscs/mkr/

**Comment by lucamar** [comment]
`.depend:887: *** multiple target patterns.  Stop.` in `/scratch/snx2000tds/jenscscs/TestingEB/tmp/eb-Gi5vR0/easybuild-LAMMPS-11Aug17-20170908.122410.mxbeB.log`

**Comment by jgphpc** [comment]
Merging because it builds fine on login nodes and we will not perftools recipes in jenkins list.

**Pull request merged**